src/elements/Icon/IconGroup.js

```diff
@@ -0,0 +1,56 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+
+/**
+ * Several icons can be used together as a group
+ */
+function IconGroup(props) {
+  const {
+    className, children, size,
+  } = props
+
+  const classes = cx('icon-group',
```

Comment:
Now that we no longer use the sd-* class, we only use SUI classes. Let's remove this "icon-group" class.